### ISSUE TITLE
NTR: quinaldine catabolic process OR quinoline methyl-derivative catabolic process

### ISSUE THREAD

**Issue by Antonialock** [geneontology/go-ontology#27606]
Please provide as much information as you can: 

* **Suggested term label:**
I'm not sure what granularity would be best for this term:

1. quinaldine catabolic process
or
2. quinoline methyl-derivative catabolic process

* **Definition (free text)**
1. The chemical reactions and pathways involving quinaldine, a methyl derivative of the heterocyclic compound quinoline
or 
2. The chemical reactions and pathways involving methyl derivatives of the heterocyclic compound quinoline

* **Reference, in format PMID:#######**
 10746195

* **Gene product name and ID to be annotated to this term**
O31266 HOD

* **Parent term(s)**
same as for GO:0019335

* **Children terms (if applicable)** Should any existing terms that should be moved underneath this new proposed term?

if (1) then none but if (2) then perhaps GO:0019335 should be merged in as a broad synonym? (there is also a 'metabolic process' term for 3-methylquinoline)

* **Synonyms (please specify, EXACT, BROAD, NARROW or RELATED)**

 2-methylquinoline catabolic process

* **Cross-references**
* For enzymes, please provide RHEA and/or EC numbers. 
EC:1.13.11.48 

also see https://biocyc.org/pathway?orgid=META&id=PWY-721

* **Any other information**
from the paper:
However,   for   the   Gram-positive   Arthrobacter   spec.   Rü61a,   quinaldine   (2-methyl-quinoline)  [1]  is  sole  source  of  carbon,  nitrogen  and  energy.  
....In  the  following  step,  HOD  cleaves  the  N-heterocycle.  This  enzyme catalyses the incorporation of molecular oxygen into positions C2 and C4 with concomitant release of carbon monoxide (BAUERet al. 1994, 1996). The formed N-acetylanthranilate [5]is hydrolysed to anthranilate [6] and is further degraded to catechol [7] (HUNDet al. 1990).

**Comment by pgaudet** [comment]
Since [3-methylquinoline catabolic process](http://purl.obolibrary.org/obo/GO_0019335) corresponds to  https://biocyc.org/pathway?orgid=META&id=PWY-721, why not rename it quinaldine catabolic process?

**Comment by pgaudet** [comment]
@edwong57 Just to clarify: The reaction labelled EC:1.13.11.- in PWY-721 would correspond to the activity @Antonialock would like to annotate, EC:1.13.11.48 ? 

I am not sure we need a new term - 'quinaldine catabolic process' could be added as a synonym to 3-methylquinoline catabolic process (GO:0019335). 

The definition of 3-methylquinoline catabolic process (GO:0019335) could be made more specific to state that 3-methylquinoline is degraded to N-acetyl-anthranilate, citing the paper that Antonia mentions (PMID:10746195)

Does that work?

**Comment by edwong57** [comment]
> [@edwong57](https://github.com/edwong57) Just to clarify: The reaction labelled EC:1.13.11.- in PWY-721 would correspond to the activity [@Antonialock](https://github.com/Antonialock) would like to annotate, EC:1.13.11.48 ?

@pgaudet, yes, I believe that reaction is the one that @Antonialock is referring to.
 
> I am not sure we need a new term - 'quinaldine catabolic process' could be added as a synonym to 3-methylquinoline catabolic process (GO:0019335).

Yeah, I'm not sure, either. Quinaldine is 2-methylquinoline (https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:132813), whereas GO: 0019335 is describing the catabolic process of 3-methylquinoline. 
 
> The definition of 3-methylquinoline catabolic process (GO:0019335) could be made more specific to state that 3-methylquinoline is degraded to N-acetyl-anthranilate, citing the paper that Antonia mentions (PMID:10746195)

In the paper, it looks like 2-methylquinoline is degraded to N-acetyl-anthranilate, not 3-methylquinoline. 

![Image](https://github.com/user-attachments/assets/668252a1-e052-4463-8965-3476a14c02eb)

> Does that work?

Since they are two different inputs, I think using GO:0019335 with the current or more specific definition wouldn't be correct.

Maybe we should go with option #2 - make a new, more general term - 'methylquinoline catabolic process' (or rename and redefine GO:0019335 to make it more general) and add 2-methylquinoline catabolic process, 3-methylquinoline catabolic process, and quinaldine catabolic process as broad synonyms.

Would that work?

**Comment by edwong57** [comment]
@pgaudet, @Antonialock 

Would making a new, more general term - 'methylquinoline catabolic process' (or rename and redefine GO:0019335 to make it more general) and adding 2-methylquinoline catabolic process, 3-methylquinoline catabolic process, and quinaldine catabolic process as broad synonyms work?

**Comment by pgaudet** [comment]
what do you think @Antonialock ?

**Comment by pgaudet** [comment]
OK. They we would make the MetaCyc xref a narrowMatch, right ?

**Comment by Antonialock** [comment]
yes I presume so, - we would be lumping different degradation pathways
